ls1intum/Artemis
# `Exam mode`: Change label of the example solution repo download button

### Checklist
#### General


#### Client


### Motivation and Context


As discussed in the thread starting with https://github.com/ls1intum/Artemis/pull/6087#discussion_r1083893184, using `Download Repo` label for the example solution download button of the exam programming exercises can be understood as downloading the student's own repository, which is misleading.

### Description

Changed label for the example solution download button of the exam programming exercises from `Download Repo` to `Download Example Solution` and updated related text and screenshot in the docs.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exam 
- 1 Student who submitted the Exam


1. Log in to Artemis
2. Navigate to Course Administration
3. As an instructor, set the example solution publication date of the Exam to a past date.
4. As a student who submitted the exam, navigate to the exam summary  and check if the button next to the programming exercise says `Download Example Solution` button instead of `Download Repo`.
5. Check that German version of the same page has no missing translations.
6. Check the button still works as before (i.e., downloads the example solution repo).

### Review Progress






#### Code Review

#### Manual Tests


### Test Coverage




Not changed.

### Screenshots


![image](https://user-images.githubusercontent.com/18427209/215261339-d929ff04-0e03-470f-ae7d-15043b1f16c8.png)
